### PR TITLE
Correct namespace prefix for inlined deps

### PR DESCRIPTION
**Pull request by dpsutton**
Before submitting a PR make sure the following things have been done:

- [x] The commits are consistent with our [contribution guidelines](CONTRIBUTING.md)
- [ ] You've added tests to cover your change(s)
- [ ] All tests are passing
- [ ] The new code is not generating reflection warnings

Apropos is broken right now and including inlined deps for this reason and another reason that will be patched on cider-nrepl and cider soon.

### CODE REVIEW

File: src/orchard/xref.clj [hunk 1]
```diff
@@ -47,7 +47,9 @@
   {:added "0.5"}
   [var]
   (let [var (as-var var)
-        all-vars (q/vars {:ns-query {:project? true} :private? true})
+        all-vars (q/vars {:ns-query {:project? true
+                                     :exclude-regexps ["^cider.nrepl.inlined-deps" "^nrepl"]}
```
Comment (bbatsov) [inline src/orchard/xref.clj]: I think we should probably put the list of exclusions in some var that we can just check where this is needed. 

I also wonder if it won't better for the query methods to just filter those out automatically.

Comment (dpsutton) [reply src/orchard/xref.clj]: The `q/namespace` function does filter it out. It just had the wrong prefix. This technically isn't needed.

Comment (dpsutton) [reply src/orchard/xref.clj]: I reset the branch to just the single commit and force pushed. no longer there.

**Comment by bbatsov** [comment]
Thanks!

**Pull request merged**